README.md

```diff
@@ -0,0 +1,5 @@
+# Sample README
+
+## a readme to explain nothing
+
+add some text here. i am going to mispell a bnunch of words to be reveiwoed later for fixess.
```

Comment:
Some words are misspelled here. Please go back and make sure everything is spelled correctly!